rtc_tenclave/src/dh/types.rs

```diff
@@ -0,0 +1,145 @@
+use secrecy::{Secret, Zeroize};
+use sgx_types::*;
+
+#[cfg(test)]
+use mockall::automock;
+
+pub struct AlignedKey(sgx_align_key_128bit_t);
```

Comment:
TODO: Add a comment that links to https://github.com/apache/incubator-teaclave-sgx-sdk/blob/master/release_notes.md#rust-sgx-sdk-v110 to indicate why we are using the align_key types.

Reply:
I agree. If you don't mind, can you please create a backlog task for creating a docs folder, and then just copy or link to this conversation as something that should be added to it?

Reply:
Added in 9af933b: how does that look?